examples/ota-requestor-app/linux/main.cpp

```diff
@@ -54,23 +54,22 @@ OTAImageProcessorImpl gImageProcessor;
 bool HandleOptions(const char * aProgram, OptionSet * aOptions, int aIdentifier, const char * aName, const char * aValue);
 void OnStartDelayTimerHandler(Layer * systemLayer, void * appState);
```

Comment:
This function and it's definition down below should be removed as well

Reply:
Done

Reply:
Done